src/main/python/keyboard_to_nt.py

```
@@ -1,33 +1,43 @@
     print("Connected!")
     table = ntcoreinst.getTable("SmartDashboard/keyboard")
 
-    keyboard.hook(lambda e: on_action(e))
-    keyboard.wait()
+    def on_press(key):
+        handle_key(key, True)
+
+    def on_release(key):
+        handle_key(key, False)
+
+    def handle_key(key, is_down):
```

Ruff (0.14.10)

Missing return type annotation for private function `handle_key`

Add return type annotation: `None`

(ANN202)